Code/src/main/java/nl/utwente/group10/ui/components/blocks/Block.java

```diff
@@ -93,8 +97,46 @@ public final CustomUIPane getPane() {
      * change.
      *
      * This method should only be called after the Block's constructor is done.
```

Comment:
Niet helemaal. De andere methoden doen eigenlijk nog steeds wat ze horen te doen met een niet geheel geldige connectie, finishConnection() geeft een foute (niet gefinishde) Connection terug als return indien je hem aanroept met een lege Connectie.

Heb het nu wat aangepast, doet nu een extra check en geeft een null terug als de Connection niet gefinished is. De waarschuwing is daarop weggehaald.